#!/usr/bin/env python
PACKAGE = "franka_interactive_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("activate_tool_compensation", bool_t, 0, "trigger to activate external tool compensation (the desired compensation forces defined in ./config)", True)
gen.add("set_locked_joints_position", bool_t, 0, "Set current joint configuration to lock joints", False)
gen.add("activate_lock_joint6", bool_t, 0, "trigger to lock/unlock joint6", False)
gen.add("activate_lock_joint7", bool_t, 0, "trigger to lock/unlock joint7", False)

exit(gen.generate(PACKAGE, "dynamic_compliance", "gravity_compensation_param"))
